python/SignaliQ/model/CloudProviderEvent.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, description, environment_id,
             "cloudVMs": [],
         }
         self.internalType = internal_type
-        self.isNoise = True
+        # self.isNoise = False ## Do not specify the isNoise flag
```

Comment:
you need isNoise = true for the third part event correlation to work (unless we're going to change backend code) -- just happened to be looking at that code today

Reply:
There should now be a check for null in CloudProviderEventCollectionTask (around line 160) that sets the value to true if not specified.

It's still ok to leave it in also.

Reply:
ok, yeah, looks like isNoise should get set in all cases (although with the unmapped possibility in there, it's not straightforward to see that that is the case)

either way, being explicit here doesn't hurt...and it makes sure we're not depending on some undocumented internal behavior (which could easily break or change...just saying)